news/_posts/core-weekly/2021-07-27-coreweekly-29-2021.md

```diff
@@ -0,0 +1,171 @@
+
+
+### stylelint configuration
+* [#19](https://github.com/PrestaShop/stylelint-config/pull/19): Bump stylelint-config-twbs-bootstrap from 2.2.2 to 2.2.3. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+* [#18](https://github.com/PrestaShop/stylelint-config/pull/18): Bump stylelint-config-twbs-bootstrap from 2.2.1 to 2.2.2. Built by [@dependabot[bot]](https://github.com/apps/dependabot)
+
+
+### Links list module
+* [#131](https://github.com/PrestaShop/ps_linklist/pull/131): Make ps_linklist compatible with twig 3, by [@atomiix](https://github.com/atomiix)
+
+
+### TranslationTools Bundle
+* [#94](https://github.com/PrestaShop/TranslationToolsBundle/pull/94): Twig compatibility (v1, v2 & v3), by [@atomiix](https://github.com/atomiix)
+
+
+### Contact Form module
+* [#56](https://github.com/PrestaShop/contactform/pull/56): Added WebP as authorized extension in Attachment, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Social Follow module
+* [#26](https://github.com/PrestaShop/ps_socialfollow/pull/26): Introduce localized links. Thank you [@lmeyer1](https://github.com/lmeyer1)
+
+
+### Customer "Sign in" link module
+* [#40](https://github.com/PrestaShop/ps_customersignin/pull/40): Deprecate variable for logged in customer. Thank you [@Hlavtox](https://github.com/Hlavtox)
+
+
+### Wishlist block module
+* [#114](https://github.com/PrestaShop/blockwishlist/pull/114): Facilitate customizations of the module, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#109](https://github.com/PrestaShop/blockwishlist/pull/109): Javascript bundles optimization. Thank you [@Oksydan](https://github.com/Oksydan)
+
+
+### Auto Upgrade module
+* [#369](https://github.com/PrestaShop/autoupgrade/pull/369): Avoid drop of some tables like guests, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: [@Progi1984](https://github.com/Progi1984), [@PierreRambaud](https://github.com/PierreRambaud), [@dependabot[bot]](https://github.com/apps/dependabot), [@sowbiba](https://github.com/sowbiba), [@matks](https://github.com/matks), [@atomiix](https://github.com/atomiix), [@eternoendless](https://github.com/eternoendless), [@jolelievre](https://github.com/jolelievre), [@nesrineabdmouleh](https://github.com/nesrineabdmouleh), [@kpodemski](https://github.com/kpodemski), [@Seb33300](https://github.com/Seb33300), [@boubkerbribri](https://github.com/boubkerbribri), [@Arnaud104](https://github.com/Arnaud104), [@okom3pom](https://github.com/okom3pom), [@lmeyer1](https://github.com/lmeyer1), [@jf-viguier](https://github.com/jf-viguier), [@Hlavtox](https://github.com/Hlavtox), [@NeOMakinG](https://github.com/NeOMakinG), [@Oksydan](https://github.com/Oksydan), [@Sinepel](https://github.com/Sinepel), [@Quetzacoalt91](https://github.com/Quetzacoalt91)!
```

Comment:
I'm listed but there are no pull requests from me merged this week 🤔

Reply:
Your PR https://github.com/PrestaShop/PrestaShop/pull/25316 was merged. But it does not appear in the list. Something is wrong.

Reply:
Found the root cause. Oh no 🤦  https://github.com/PrestaShop/prestashop.github.io/issues/936

Reply:
Fixed https://github.com/PrestaShop/prestashop.github.io/pull/935/commits/c5b3936130e656ebd8cfdb2e3b7ca0b79c8e9d58